core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -517,4 +517,11 @@ public void applyTo(ViewMetadata.Builder viewMetadataBuilder) {
       viewMetadataBuilder.setCurrentVersionId(versionId);
     }
   }
+
+  class EnableRowLineage implements MetadataUpdate {
+    @Override
+    public void applyTo(TableMetadata.Builder metadataBuilder) {
+      MetadataUpdate.super.applyTo(metadataBuilder.enableRowLineage());
```

Comment:
shouldn't this be just `metadataBuilder.enableRowLineage()` without the super call?

Reply:
this also requires changes to `MetadataUpdateParser` to properly pass the flag

Reply:
and we'd need to update/vote on the REST spec changes for this change once we add it to `MetadataUpdateParser`

Reply:
Yeah i'm not sure what happened here, I don't remember writing this, must have been Intellij auto refactoring something when I dragged it around.